pyext/src/output.py

```diff
@@ -773,7 +862,7 @@ def get_fields(self, fields, filtertuple=None, filterout=None, get_every=1):
             #if line_number % 1000 == 0:
             #    print "ProcessOutput.get_fields: read line %s from file %s" % (str(line_number), self.filename)
             try:
-                d = ast.literal_eval(line)
+                d = eval(line)
             except:
```

Comment:
Never use a bare `except:`. Catch a specific range of exceptions, e.g. `except ValueError:`.